twitchio/message.py

```diff
@@ -61,6 +64,15 @@ def __init__(self, **kwargs):
         self._tags = kwargs.get("tags")
         self.echo = kwargs.get("echo", False)
 
+        if self._tags is not None:
+            self.first = self._tags.get("first-msg")
+            if self.first == "1":
+                self.first = True
+            elif self.first == "0":
```

Comment:
```suggestion
            else:
```
This should default to false if the tag is an unknown value (as this is the default 99% of the time)